packages/calypso-products/src/plans-list.tsx

```diff
@@ -3050,6 +3051,7 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 			PRODUCT_JETPACK_BOOST,
 			PRODUCT_JETPACK_SOCIAL_ADVANCED,
 			PRODUCT_JETPACK_SEARCH,
+			PRODUCT_JETPACK_STATS_MONTHLY,
```

Comment:
This will be later changed to Stats yearly

Reply:
Looks like it just got merged 😄